content/posts/2025-vision.md

```diff
@@ -0,0 +1,97 @@
+title: Read the Docs 2025 Vision: great support for all documentation tools
+date: 2025-01-14
+description: Our vision for Read the Docs in 2025, supporting all documentation tools and enhancing frontend features with Addons.
+category: Meta
+tags: addons, vision, builders
+authors: Eric Holscher
+status: published
+image: /images/2025-vision.jpg
+image_credit: Photo by <a href="https://unsplash.com/@mcnoble?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Matt Noble</a> on <a href="https://unsplash.com/photos/selective-focus-photo-of-silver-tower-viewer-telescope-facing-sunshine-BpTMNN9JSmQ?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+As we look forward to 2025,
+we took some time to reflect on where we've been and where we're going,
+and wanted to share that with you.
+
+**Our vision for Read the Docs is to be the best place to host software documentation using a docs-as-code approach**.
+This means that we're building on top of Git,
+and providing features that make sense for software documentation.
+
+**Our product philosophy is to provide a great default experience but allow users to customize and extend as needed.**
+This means that we provide a set of features that work for most projects,
+but then allow you to extend or replace them as needed.
+
+Below is a breakdown of how that plays out with each of the major parts of Read the Docs.
+
+## Support for all documentation tools
+
+We've been working to expand build support beyond Sphinx and MkDocs,
+and build Addons for powerful frontend features that work on any tool.
+
+The major steps here have been:
+
+* **Fully redesigned dashboard:** We've been working on a [new dashboard](https://about.readthedocs.com/blog/2024/11/rollout-of-our-new-dashboard/) that brings our site into the modern era. This has been fully rolled out to all users as the default, and we're continuing to improve it.
```

Comment:
This feels worth calling out but maybe in a different section. This doesn't seem like it speaks to "We've been working to expand build support beyond Sphinx .." above.

Reply:
Yea, I kinda shoe-horned it in because it seemed notable, but perhaps a larger "improved UX" section would be good.

Reply:
I added it to the Addons section and refocused that as a larger UX approach.